samples/unity-of-bugs/Assets/Resources/Sentry/link.xml

```diff
@@ -0,0 +1,65 @@
+﻿<linker>
+    <assembly fullname="Sentry" preserve="all"/>
+    <assembly fullname="Sentry.Unity" preserve="all"/>
+    <assembly fullname="System.Text.Json" preserve="all"/>

```

Comment:
We will for sure need to fine tune this but for now it's OK
https://github.com/getsentry/sentry-unity/issues/81

Reply:
Sure.